doc/man3/EVP_set_default_properties.pod

```diff
@@ -27,10 +30,22 @@ given with I<libctx> (NULL signifies the default library context).
 Any previous default property for the specified library context will
 be dropped.
 
+EVP_default_properties_enable_fips() sets the 'fips=yes' to be a default property
+if I<enable> is non zero, otherwise it clears 'fips=yes' as a default property
```

Comment:
`s/as a/from the/`
`s/$/query/`

Note: the code above does **not** clear the 'fips=yes' setting, it adds 'fips=no' which is quite different.